### ISSUE TITLE
Any progress? Deprecation pending

### ISSUE THREAD

**Issue by dy** [audiojs/audio-mpg123#20]
Hi @jamen @connorhartley!
How are you doing?
I'm cleaning up the org and deprecating stuff from npm, many audiojs things are useless and overcomplication.
Please either scrapjs this package or archive. node-speaker authors seem to have power and qualification to maintain their stuff, not us.

**Comment by jamen** [comment]
Hey! I've been good. How about you two?

Coincidentally I just cloned this. Not sure if I pinged you somehow lol. I felt like revisiting audio in node.js after gaining some experience with Rust. I was thinking of using the [`cpal`](https://crates.io/crates/cpal) library.

I agree with archiving some stuff in audiojs. I think `web-audio-api` and `audio` should be kept at minimum. Since a lot of the repos are yours I don't mind you making the call on it. If I do anyhting, it'll be in the `audio`, `TooTallNate/speaker`, or `web-audio-api`  projects.

What I'd like to see is merging all of these audio functions into the `audio` library , which it looks like you did to some degree (just skimming the code, it's been a while). If I start doing native code again I'll put it in the `audio` library.

Could you reinvite me to audiojs and scrapjs? Personally I like archiving repos on my profile now, so I might go through and transfer back some stuff I did (i'll leave anything that was collaborative and not mine of course)

**Comment by dy** [comment]
Sure, just invited you again.

As for web-audio-api - it's considered hairy API, I can see options:
1. either finishing it as is, making minimal workable thing (going to be a significant effort)
2. reusing [web-audio-engine](https://www.npmjs.com/package/web-audio-engine) - that seems to be more complete
3. building on something very simple and trivial like extended [zzfx](https://ghub.io/zzfx) (I'm working on that tool, got PoC ready - feels amazing, although code generation is slow-ish)
4. just archiving the project, we're not qualified for really optimal implementation, there needs to be a serious sponsor and full-time engagement, just check @chrisguttandin work at [standardized-audio-context](https://github.com/chrisguttandin/standardized-audio-context)

I want to leave in this org only workable tools.
I wouldn't like definitely:
- spreading API by packages, npm proved to create unnecessary complication and bureaucratic barriers
- maintaining/running uncool/fake/dumb/never finished packages
- mirroring some known libs, like mpg-123/cubeb/etc (fake/unmaintained mirror is rather burden and noise for everyone)

↑ that stuff forms hatred towards the org and us maintainers.

**Comment by dy** [comment]
@jamen I think if you decide to start off a new alternative speaker implementation, let's keep it private until we can publish at-least PoC

**Comment by jamen** [comment]
@dy yeah thats my plan. i wasn't even going to say anything, but you happened to reach out the same day lol

**Comment by jamen** [comment]
Since this and audiojs/mpg123-mirror are @connorhartley's he should decide where they go. I'll take node-cubeb and audio-decode-wasm

**Comment by vectrixdevelops** [comment]
Hey, I've been well.

I definitely agree mpg123 is not realistically/easily maintainable for anyone who isn't dedicated to keep these up to date. I think there are much better solutions (possibly like the one @jamen is working on) for such a project. I can archive and take these two projects.
